tools/OresToFieldGuide/MainClass.cs

```diff
@@ -1,4 +1,11 @@
-﻿// See https://aka.ms/new-console-template for more information
+﻿//Uncomment this to enable verification of vein weights
+#define VERIFY_VEIN_WEIGHTS
+//Uncomment this to write json in a Prettified format, only really useful for debugging the output
+//#define PRETTY_PRINT
```

Comment:
keep this enabled by default, we want json to be easily mergeable, not compressed